### ISSUE TITLE
Research methods for conducting large-scale image classification through embedding.

### ISSUE THREAD

**Issue by UmbertoFasci** [UmbertoFasci/Whale_Dolphin_Identification_Competition#1]
- [x] Read the paper [Information-theoretical label embeddings for large-scale image classification](https://arxiv.org/pdf/1607.05691.pdf)

**Comment by UmbertoFasci** [comment]
## Paper Overview
This paper seems useful at it presents a method for training multi-label, multi-class image classification models that is faster and more accurate than supervised logistic regression. This method consists of embedding high-dimensional sparse labels onto a lower-dimensional dense sphere of vectors where the classification problem can then be treated as a **cosine proximity regression** problem on this sphere.

**Comment by UmbertoFasci** [comment]
## Embedding High-Dimensional Labels
Instead of focusing on the capture of visual similarity this method focuses on the concept of visual co-occurrence of labels. The core idea of this approach is to develop a label embedding space which can capture the likelihood that any two labels would co-occur in an image.

**Comment by UmbertoFasci** [comment]
- [x] Explore TensorFlow Core v2.8.0 API to apply this general method.

**Comment by UmbertoFasci** [comment]
- [ ] Explore how Deep Siamese Convolutional Networks can be used for this Happywhales data.
